notebooks/lerobot_act/lerobot-act.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    VENV_DIR = pathlib.Path('.py310_venv')
</pre><p>Why should we create a separate venv for this notebook? I cannot run this notebook with python3.12/Windows, and looks python3.10 is not the only one that lerobot can support <a href="https://github.com/huggingface/lerobot?tab=readme-ov-file#installation" target="_blank">https://github.com/huggingface/lerobot?tab=readme-ov-file#installation</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3111/discussion/#comment-2585736592'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>